packages/@react-stately/tooltip/src/useTooltipTriggerState.ts

```diff
@@ -75,7 +78,13 @@ export function useTooltipTriggerState(props: TooltipTriggerProps): TooltipTrigg
   };
 
   let hideTooltip = () => {
-    close();
+    if (!closeTimeout.current) {
+      closeTimeout.current = setTimeout(() => {
+        closeTimeout.current = null
+        close()
+      }, TOOLTIP_COOLDOWN)
```

Comment:
I've chosen to reuse `TOOLTIP_COOLDOWN` which is equal to 500 but it might be worth shortening this timeout here, thoughts?

Reply:
I think we can probably make it a prop to pass to useTooltipTriggerState if other libraries need to control that, but we can start with this default. For the record, it's not specified in Spectrum either right now. https://spectrum.adobe.com/page/tooltip/#Behaviors